.changeset/moody-knives-check.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": minor
+---
+
+Add `categoryToUrlParam()` and `urlParamToCategory()` conversion methods
```

Comment:
We need to improve the naming if they're part of the public API. How about `pageTreeNodeCategoryToUrlParam()` and `urlParamToPageTreeNodeCategory()`?